src/components/BudgetKeyFooterComponent.ts

```diff
@@ -1,21 +1,21 @@
 import {Component} from '@angular/core';
+import {ThemeService} from '../services/ThemeService';
 
 @Component({
     selector: 'budgetkey-footer',
     template: `
         <footer class="footer">
             <div class="inner-footer">
                 <div class="inner-text">
-                    <div>מפתח התקציב</div>
-                    <div>מבית <a href="{{ hasadna_url }}">הסדנא לידע ציבורי</a></div>                
+                    <div>{{theme.siteName}}</div>
+                    <div>מבית <a href="{{hasadnaUrl}}">הסדנא לידע ציבורי</a></div>                
                 </div>
             </div>   
         </footer>
     `
 })
 
 export class BudgetKeyFooterComponent {
-
-    title = "מפתח התקציב, מבית";
-
+    public hasadnaUrl = 'http://www.hasadna.org.il/';
```

Comment:
@OriHoch, WDYT - can this also be moved to theme config?

Reply:
currently it's the same for both themes.. and I expect it will remain like that if we have other themes - because we are all related to hasadna..